elide-integration-tests/src/test/java/com/yahoo/elide/initialization/IntegrationTest.java

```diff
@@ -121,6 +124,9 @@ protected final Server setUpServer() throws Exception {
         servletContextHandler.setContextPath("/");
         server.setHandler(servletContextHandler);
 
+        servletContextHandler.setAttribute(AsyncIntegrationTestApplicationResourceConfig.ASYNC_EXECUTOR_ATTR, Executors.newFixedThreadPool(5));
+        servletContextHandler.setAttribute(AsyncIntegrationTestApplicationResourceConfig.STORAGE_DESTINATION_ATTR, Files.createTempDirectory("asyncIT"));
```

Comment:
Ideally, this would be done in a subclass of IntegrationTest.  Maybe there can be a function that manipulates the servletContextHandler that subclasses can override.